packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -149,7 +149,6 @@ export type LocalTracks = {
     audio?: LocalSystemAudioTrack;
     video?: LocalDisplayTrack;
   };
-  annotationInfo?: AnnotationInfo;
```

Comment:
we need to also remove the declaration of annotationInfo from the Meeting object's properties - line 551